index.html

```diff
@@ -10,54 +10,62 @@
   <title>My Portfolio</title>
 </head>
 <body>
+  <div class="header-bg">
+    <header>
+      <nav class="navbar">
+        <a href="">Portfolio</a>
+        <a href="">About</a>
+        <a href="">Contact</a>
+      </nav>
+      <div id="union">
+        <img src="images/Icons/Icon - Menu.png" alt="union">
+      </div>
+    </header>
 
-  <header>
-    <nav id="union">
-      <img src="images/Icons/Icon - Menu.png" alt="union">
-    </nav>
-
-  </header>
+    <section class="headline">
 
-  <section class="headline">
+      <div class="headline-text">
+        <h1>
+          Hey there.
+          <br>
+          I’m Didy.
+          <br>
+          <span class="green">I’m a Full Stack developer from Microverse</span>
+        </h1>
```

Comment:
- [ ] Please make sure your `main heading `looks just like the one in the figam. I suggest increasing `font-weight `and removing `br ` element from the heading in the desktop version. You can also divide the main heading in to multiple heading tags i.e. `hx` instead of using span in one `h1`

Figma design:

<img width="346" alt="Screenshot 2023-03-31 at 9 26 49 AM" src="https://user-images.githubusercontent.com/44798044/229028704-04fe8b25-1c15-4353-936d-9c2ab5fefe2b.png">

Your design:

<img width="958" alt="Screenshot 2023-03-31 at 9 26 55 AM" src="https://user-images.githubusercontent.com/44798044/229028666-80ab00ff-53b8-4fe6-9684-83e90d8f2432.png">